docs/src/getting_started/core.md

```diff
@@ -157,12 +162,28 @@ well as our public types from the shared library.
 Now we are in a position to create a basic app in `/shared/src/app.rs`. This is
 from the
 [simple Counter example](https://github.com/redbadger/crux/blob/master/examples/simple_counter/shared/src/counter.rs)
-(which also has tests, although we're not showing them here):
+(which also has tests, although we're not showing them here).
 
 ```rust,no_run,noplayground
+// /shared/src/app.rs
 {{#include ../../../examples/simple_counter/shared/src/app.rs:app}}
 ```
 
+```admonish
+The example uses the new [`effect!`](https://docs.rs/crux_macros/latest/crux_macros/macro.effect.html) macro, which is a bit more ergonomic than the
+`Effect` and `Export` derive macros you may have used on a `Capabilities` type in the past.
+Inside the body of the `effect!` macro invocation we simply declare an enum to represent our effects. The enum has a variant for each effect, which carries the [`Operation`](https://docs.rs/crux_core/latest/crux_core/capability/trait.Operation.html) type.
+
+The macro generates code for:
+- an _actual_ effect enum (where the inner `Operation` type is wrapped with [`crux_core::Request`](https://docs.rs/crux_core/latest/crux_core/struct.Request.html))
+- an enum, similar to the one we specify, for use across the FFI boundary,
+- an implementation of the `Effect` trait
+- some useful test helpers
+- code to support the current foreign type generation (that uses `serde-reflection`)
+
+You can also use the `effect!` macro with the [new typegen](https://github.com/redbadger/crux/pull/217).
+```
+
```

Comment:
Since this is a Getting Started, should we just speak to the way people should be doing it now, if they're coming to it fresh? So that means commenting on the reason there is a `Capabilities` associated type (tail end of migration), rather than pointing out the `effect!` macro, I think?

Reply:
Good point. Added in several places.